### DISCUSSION TITLE
Texte multiligne dans un contrôle de pièce jointe

### DISCUSSION THREAD

**Discussion by billo04** [MTESSDev/FRW#59] [Annonces]
Dans le contrôle d'ajout de pièce jointe, est-il possible d'y placer un champ texte multiligne dans lequel l'utilisateur viendrait inscrire un commentaire relatif à la pièce qu'il joint?

Cela pourrait être utile notamment dans le cas où le type de pièce à joindre est "Autre". L'utilisateur pourra alors précisé ce qu'est ce "autre" (bail, permis de conduire, facture).

Exemple:
![image](https://user-images.githubusercontent.com/124705516/231848507-2974ea36-a173-4278-89d2-9d339f0d051b.png)

**Comment by j0ber** [comment]
Bonjour,

Non ce n'est pas possible. L'idée était justement de faire comme vous avez fait, i.e. Mettre le contrôle de pièce jointe (et autres contrôles requis) dans un groupe.

**Reply by anisite** [reply]
Si votre besoin est de faire comme sur l'image en exemple c'est possible oui, en fait tous les champs situés autour d'un champ pièce jointes sont automatiquement considérer comme des métadonnées, ça s'applique aussi aux champs cachés `hidden` 

En sortie les autres valeurs seront donc ajoutés en meta données à la pièce jointe et pourront être conservées dans une GED.

Parcontre pas possible de les mettre dans la zone en gris du contrôle pièce jointe.

Est-ce que j'ai mal compris?

**Comment by billo04** [comment]
Oui, tu as bien compris la question.

Il n'est pas souhaité que le champ texte fasse partie de la zone grise de la pièce jointe. Il est suffisant de l'avoir dans le même groupe/encadré que la pièce jointe.

**Comment by anisite** [comment]
Dans ce cas, normalement tout est ici : https://formulaires.it.mtess.gouv.qc.ca/fr/Form/7/p700u/0/N/#p=sectionAjouterPiecesJointes

il y a un exemple `Ajouter au moins un fichier et maximum 5 fichiers avec une catégorie définie par défaut` qui montre l'ajout d'une métadonnée `hidden`

Aussi plus bas, l'exemple `Ajouter maximum 5 pièces jointes` illustre comment ajouter une liste déroulante de `Type de pièce` mais ça pourrait être n'importe quel type de champ (textarea ou autres) que ce soit avant ou après le contrôle PJ, l'important c'est que ce soit dans le même groupe. `group` ou `repeatableGroup`
